index.js

```diff
@@ -16,10 +16,11 @@ const pkg = require('./package')
 module.exports = Client
 
 const flush = Symbol('flush')
-const metadataSym = Symbol('metadata')
-const transactionSym = Symbol('transaction')
-const spanSym = Symbol('span')
-const errorSym = Symbol('error')
+
+Client.metadataEnc = Symbol('metadata')
+Client.transactionEnc = Symbol('transaction')
+Client.spanEnc = Symbol('span')
+Client.errorEnc = Symbol('error')
```

Comment:
What about something like this?

```js
const encoding = {
  metadata: Symbol('metadata'),
  transaction: Symbol('transaction'),
  span: Symbol('span'),
  error: Symbol('error')
}
Client.encoding = encoding
```

Less repetitive and more clear, in my opinion, than `*Enc`.

Reply:
I'd probably also `Object.freeze` the object and upper-case names to indicate constants.

Reply:
Fixed